content/SCALE/Network/_index.md

```diff
@@ -4,10 +4,328 @@ geekdocCollapseSection: true
 weight: 60
 ---
 
+{{< toc >}}
+
 The SCALE Network section contains network configuration and settings options for active *Interfaces*, *Static Routes*, and the *Global Configuration*. 
 
 The Network screen also displays *OpenVPN* information and *IPMI* channels.
 
 ![NetworkSCALE](/images/SCALE/NetworkSCALE.png "SCALE Network Page")
 
```

Comment:
Rough suggestion to transition away from article intro to describing content in the tabs box.
```suggestion

Each Networking configurable is a separate widget on the overview screen. Click the buttons or an existing entry in a widget to slide out a side panel with configuration options. Select a networking section in the box below to see more details about the specific configuration options.
```

Reply:
Done.